### DISCUSSION TITLE
AI, National Digital Identity, and Wallets

### DISCUSSION THREAD

**Discussion by guru-barbes** [trustoverip/tswg-authentic-ai#4] [General]
I'd like to start a discussion about the intersection of AI, NDI, and wallets.

As NDI picks up pace in Asia, and Europe, we will see a wave of digital wallets and digital identities.

I'd like to discuss how AI will fit into this, from the perspective of identity and wallet architects and providers:

* How will AI be built into solutions and services that will come into the market?
* How will AI be used to try to hack these solutions and services?

**Comment by wenjing** [comment]
NDI = national digital identity ?

**Reply by guru-barbes** [reply]
Yes, sorry - National Digital Identity. That's how it's being referred to in places like Singapore, Malyasia, Indonesia, Bhutan...
